pom.xml

```diff
@@ -1277,7 +1277,7 @@
               <exclude>**/.settings/**/*</exclude>
 
               <!-- Proto/grpc generated wrappers -->
-              <exclude>**/sdks/python/apache_beam/runners/api/*.py</exclude>
+              <exclude>**/apache_beam/runners/api/*_pb2*.py</exclude>
```

Comment:
Actually it makes sense to move api only, as transforms depend on it and it is not a runner. But portability is another runner implementation and it can stay where it is.